dot/types/inherents.go

```diff
@@ -46,29 +45,8 @@ func (d *InherentsData) String() string {
 	return str
 }
 
-// SetInt64Inherent set an inherent of type uint64
-func (d *InherentsData) SetInt64Inherent(key []byte, data uint64) error {
-	if len(key) != 8 {
-		panic(fmt.Sprintf("inherent key must be 8 bytes but is: %v", key))
-	}
-
-	val := make([]byte, 8)
-	binary.LittleEndian.PutUint64(val, data)
-
-	venc, err := scale.Marshal(val)
-	if err != nil {
-		return err
-	}
```

Comment:
when comparing with the substrate code I noticed that when we try to encode `InherentData` in substrate with inherents added with key `timstap0` where the value is of type `i64` that the encoding differs.  I ended up added a test in `inherents_test.go` to ensure the same encoding.